tools/c7n_azure/c7n_azure/resources/mysql.py

```diff
@@ -107,3 +108,32 @@ def __call__(self, resource):
             resource['properties'][key] = query.serialize(True).get('properties')
 
         return super().__call__(resource['properties'][key])
+
+
+@MySQL.filter_registry.register('mysql-server-security-alert-policies-filter')
```

Comment:
rename to remove the redundancies,  `security-alert`

Reply:
renamed